docs/src/pages/[platform]/components/message/examples/MessageThemeExample.tsx

```diff
@@ -0,0 +1,46 @@
+import { Flex, Message, Theme, ThemeProvider } from '@aws-amplify/ui-react';
+
+const theme: Theme = {
```

Comment:
So that this code works in JSX too and so TS doesn't complain, you can do this instead:
```suggestion
const theme = createTheme({
```

Reply:
Ohhh right, thanks. fixed in latest